packages/cli-helpers/src/lib/project.ts

```diff
@@ -33,21 +36,113 @@ export const getInstalledRedwoodVersion = () => {
   }
 }
 
-export const addEnvVarTask = (name: string, value: string, comment: string) => {
+/**
+ * Updates the project's redwood.toml file to include the specified packages plugin
+ *
+ * Uses toml parsing to determine if the plugin is already included in the file and
+ * only adds it if it is not.
+ *
+ * Writes the updated config to the file system by appending strings, not stringify-ing the toml.
+ */
+export const updateTomlConfig = (packageName: string) => {
+  const redwoodTomlPath = getConfigPath()
+
+  const configLines = []
+
+  const configContent = fs.readFileSync(redwoodTomlPath, 'utf-8')
```

Comment:
We do have a `getRawConfig` in `project-config` which will read the toml, perform env var interpolation but does not include our defaults. Would this be useful here?

Reply:
I didn't know about `getRawConfig` and its `@iarna/toml` use. The `toml` package parsed much nicer so it could look deeper. With `getRawConfig` get squiggles:

![image](https://github.com/redwoodjs/redwood/assets/1051633/948013de-4272-42b5-ab3e-c70ed030f247)

But, I can rewrite so we reuse the helpers.